Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -1143,4 +1154,55 @@ def problem(text)
       @problems << text
     end
   end
+
+  class TapAuditor
+    attr_reader :name, :path, :tap_audit_exceptions, :problems
+
+    def initialize(tap, options = {})
+      @name                 = tap.name
+      @path                 = tap.path
+      @tap_audit_exceptions = tap.audit_exceptions
+      @strict               = options[:strict]
+      @problems             = []
+    end
+
+    def audit
+      audit_json_files
+      audit_tap_audit_exceptions
+      self
+    end
+
+    def audit_json_files
+      Dir[@path/"**/*.json"].each do |file|
```

Comment:
```suggestion
      Dir[@path/"audits/*.json"].each do |file|
```

or something similar.

Reply:
The idea here is to check _all_ json files in the Tap directory. This includes the `audit_exceptions/*` files as well as the files in the root directory (like `formula_renames.json` and `tap_migrations.json`)

Reply:
I don't think we should check all JSON files. There's no reason to concern ourselves with JSON files that Homebrew doesn't own/control.

Reply:
What files would this check that Homebrew doesn't control?

Reply:
Any JSON files in a tap that we don't read from but may be being used for any other reasons.

Reply:
Okay, how about adding a list of homebrew-controlled JSON files to check?

(Ironic, adding a list to this repo in a PR trying to remove the lists 😄)

Reply:
> Okay, how about adding a list of homebrew-controlled JSON files to check?

Sounds good!